### PR TITLE
Mct/bing v7

### PR DESCRIPTION
**Pull request by MothOnMars**

### CODE REVIEW

File: app/engines/bing_image_response_parser.rb [hunk 1]
```diff
@@ -9,36 +9,35 @@ def total
 
   private
 
+  def default_bing_response_parts
+    super.merge({
+      next_offset: nil,
+      total_estimated_matches: 0,
+      value: [],
+    })
```
Comment (peggles2) [inline app/engines/bing_image_response_parser.rb]: Super picky but, no comma for last value

File: app/engines/bing_formatted_query.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-class AzureFormattedQuery < FormattedQuery
+class BingFormattedQuery < FormattedQuery
   DEFAULT_DOMAIN_SCOPE = 'site:gov OR site:mil'.freeze
```
Comment (peggles2) [inline app/engines/bing_formatted_query.rb]: Is this replacing AzureFormattedQuery? I still see it in coverage/index.html line 43909.

File: app/engines/bing_image_response_parser.rb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-class BingV6ImageResponseParser < BingV6ResponseParser
+class BingImageResponseParser < BingResponseParser
   def results
```
Comment (peggles2) [inline app/engines/bing_image_response_parser.rb]: BingV6ImageResponseParser is also being referenced in : coverage/index.html line 43958 and 44265

File: app/engines/bing_response_parser.rb [hunk 4]
```diff
@@ -1,4 +1,4 @@
-class BingV5ResponseParser
+class BingResponseParser
   attr_reader :engine
```
Comment (peggles2) [inline app/engines/bing_response_parser.rb]: This is also being referenced in index.html

File: app/engines/bing_formatted_query.rb (hunk 2 above)
Comment (MothOnMars) [reply app/engines/bing_formatted_query.rb]: Sounds like that's from a  previous test run. If you nuke your `coverage` directory and re-run the specs, you should not see those old lines in `coverage/index.html`. (Searching with `git grep` should also ignore the `coverage` directory, as it's listed in `.gitignore`.)

**Comment by MothOnMars** [comment]
@peggles2 , I added two commits yesterday to add some extra test coverage:
https://github.com/GSA/search-gov/pull/144/commits/572352d3a348ed4c2d8fa99e8b9b52088b690c14
https://github.com/GSA/search-gov/pull/144/commits/ebe6d04a2a65a8f9776be8c741a6b0fdab3cec40

Once you've had a chance to review & approve those commits, I will deploy for testing.

**Pull request merged**